src/theme.ts

```diff
@@ -83,6 +83,12 @@ const theme = createMuiTheme({
         body: {
           backgroundColor: '#05699b',
         },
+        h6: {
```

Comment:
Is this just a one-off style, or have you noticed a consistent pattern where we should override this in the theme? If we are using this everywhere then go for it. But if it's only in the ContactDetailsTab, maybe we should just make a custom style/class there?

Reply:
@TheNoodleMoose this is the Heading 6 style in the Figma doc.

Reply:
I agree with @OzzieOrca , if we want to use the preset typography that MUI provides us, this is the wrong place to specify the "h6" styles